components/upload/demo/picture-card.md

```diff
@@ -103,7 +103,7 @@ const App: React.FC = () => {
       >
         {fileList.length >= 8 ? null : uploadButton}
       </Upload>
-      <Modal open={previewOpen} title={previewTitle} footer={null} onCancel={handleCancel}>
+      <Modal visible={previewOpen} title={previewTitle} footer={null} onCancel={handleCancel}>
```

Comment:
open is correct. https://ant.design/changelog-cn#4.23.0

Reply:
OK,thanx